src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/SqlColumnEncryptionAzureKeyVaultProvider.cs

```diff
@@ -242,7 +242,6 @@ public override byte[] EncryptColumnEncryptionKey(string masterKeyPath, string e
         {
             // Validate the input parameters
             ValidateNonEmptyAKVPath(masterKeyPath, isSystemOp: true);
-            ValidateNotNullOrWhitespace(encryptionAlgorithm, nameof(encryptionAlgorithm));
```

Comment:
`ValidateEncryptionAlgorithm()` already validates if `encryptionAlgorithm` is null or not equal to "RSA_OAEP".